modules/ServerAPI/src/server/routes/ServerUtilityFunctions.coffee

```diff
@@ -40,10 +40,10 @@ exports.promisifyRequestResponseStatus = (fun, args) ->
 	# Custom promisify function to convert "json, status" type callback
 	# functions where status > 400 is an error to a standard err, response
 	# promise return
-	fun[util.promisify.custom] = (args) =>
+	fun[util.promisify.custom] = (...args) =>
```

Comment:
This function essentially takes our standard ACAS proxy routes which use the function callback signature "json, status" and turns it into a promise.  It makes code that uses these functions much easier.  Previously this function didn't support signatures that had more than one argument like the setupRoutesInternal function added in this commit.